server/charts/historian/templates/historian-configmap.yaml

```diff
@@ -37,12 +37,55 @@ data:
             "endpoint": "{{ .Values.historian.error.endpoint }}"
         },
         "throttling": {
-            "maxRequestsPerMs": {{ .Values.historian.throttling.maxRequestsPerMs }},
-            "maxRequestBurst": {{ .Values.historian.throttling.maxRequestBurst }},
-            "minCooldownIntervalInMs": {{ .Values.historian.throttling.minCooldownIntervalInMs }},
-            "minThrottleIntervalInMs": {{ .Values.historian.throttling.minThrottleIntervalInMs }},
-            "maxInMemoryCacheSize": {{ .Values.historian.throttling.maxInMemoryCacheSize }},
-            "maxInMemoryCacheAgeInMs": {{ .Values.historian.throttling.maxInMemoryCacheAgeInMs }}
+            "restCallsPerTenant": {
+                "generalRestCall": {
+                    "maxPerMs": {{ .Values.historian.throttling.restCallsPerTenant.maxPerMs }},
```

Comment:
it should be restCallsPerTenant.**generalRestCall**.maxPerMs

Same for others.

Reply:
Got it. Thanks for good catch!